### ISSUE TITLE
how to use parameter combiner in function column_embedding()

### ISSUE THREAD

**Issue by fzhang612** [rstudio/tfestimators#145]
I am having some difficulties to understand how to use the combiner parameter in function column_embedding() from the help document.

Let's say i have a simple 3 row data frame `data_frame( x = c('a', 'b', 'a b') )`. If I want to embed variable `x` into a 2 dimension vector, I will do `column_embedding( column_categorical_with_vocabulary_list( 'x', vocabulary_list = c('a', 'b'), dimension = 2, combiner = 'mean'  )`. In this case I think I will get embedding for `'a'` in the first row, embedding for `'b'` in the second row, and the mean of embeddings of `'a'` and `'b'` in the 3rd row. Is it correct? If it is not, what should I do to achieve my goal?

Thanks!

**Comment by terrytangyuan** [comment]
Yes your understanding is correct. This might be helpful as well: https://www.tensorflow.org/versions/master/get_started/feature_columns#indicator_and_embedding_columns

**Comment by kevinykuo** [comment]
Wouldn't the `'a, b'` be out of vocabulary in this case? Does `column_categorical_*` do tokenization?

**Comment by terrytangyuan** [comment]
Sorry yea that would be out of vocabulary. It doesn't do tokenization. The documentation for combiner is actually more detailed in [embedding lookup sparse](https://www.tensorflow.org/versions/master/api_docs/python/tf/nn/embedding_lookup_sparse) which is what embedding column is based on.

**Comment by fzhang612** [comment]
thanks, guys! 

After running thru some dummy data I can confirm the previous code doesn't work as expected.

I have to admit that [embedding lookup sparse](https://www.tensorflow.org/versions/master/api_docs/python/tf/nn/embedding_lookup_sparse) is a bit abstract to grasp. Hope to see some concrete examples in R to demonstrate how this can be done
